wrappers/kn/src/nativeTest/kotlin/Test.kt

```diff
@@ -0,0 +1,204 @@
+import zxingcpp.*
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+expect fun loadImage(path: String): ImageView
+
+class BarcodeReaderTest {
+
+	private val barcodeReader = BarcodeReader()
+
+	val singleExamples = mapOf(
+		"qr_100_hello.png" to BarcodeImpl(
```

Comment:
Please don't add additional images inside the wrapper folder. There are plenty in the `/test/samples/` directory.

Reply:
fixed in https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/6e61fbf9463f9c50adae3fdc8c1ef89de7664c8c